### ISSUE TITLE
small error with rkill?

### ISSUE THREAD

**Issue by KingZee** [bmrf/tron#154]
```
2018-11-09 23:29:55.73    Launch job 'rkill'...
[...]
Whitelist file: C:\...\tron\resources\stage_0_prep\\stage_0_prep\rkill\rkill_process_whitelist.txt not found.
```
Looks like extra '\stage_0_prep\' in rkill whitelist directory path variable?

**Comment by vocatus** [comment]
Hi @KingZee , which version of Tron did this happen on?

**Comment by vocatus** [comment]
Hi @KingZee, I've fixed the path bug and the fix will go out in the next release (10.7.3). Thanks for letting me know.
